pypeit/spectrographs/ldt_deveny.py

```diff
@@ -268,17 +269,19 @@ def default_pypeit_par(cls):
 
         # Turn off illumflat unless/until we can deal properly with flexure in
         #   the spatial direction.  All other defaults OK (as of v1.7.0)
-        set_use = dict(use_illumflat=False)
-        par.reset_all_processimages_par(**set_use)
+        #   Also, use an order=1 polynomial for fitting the overscan rather
+        #   a SavGol filter -- more appropriate for this CCD.
+        set_procpars = dict(use_illumflat=False, overscan_method='polynomial', overscan_par=1)
+        par.reset_all_processimages_par(**set_procpars)
 
         # For processing the arc frame, these settings allow for the combination of
         #   of frames from different lamps into a comprehensible Master
         par['calibrations']['arcframe']['process']['clip'] = False
         par['calibrations']['arcframe']['process']['combine'] = 'mean'
-        par['calibrations']['arcframe']['process']['subtract_continuum'] = True
+        # par['calibrations']['arcframe']['process']['subtract_continuum'] = True
```

Comment:
Is there a reason to uncomment this for the arc lines? To be fair, it's only needed in cases where arc frames are being that contain different lamp spectra. Is this the case? Looking at the dev suite, it seems that dv7 contains multiple different lamp spectra?

Reply:
The continuum-subtraction is an aesthetic issue for me -- without the subtraction (and hence negative flux values), the WaveCalib QA plots are done with a log scale.  For most of our users, the desired lamps are all on at the same time (_e.g._, HgCdAr) -- but some insist on taking separate frames for each lamp.  I have instructions in our instrument manual that if arcs with individual arc lamps are used/combined, then this parameter needs to be set to `True`.

I should check the DV7 `.pypeit` file in the DevSuite -- thanks for flagging that.

Reply:
Note: I fixed the DV7 `.pypeit` file in pypeit/Pypeit-development-suite#296.

Reply:
Just FYI - I ran the dev suite, and got an error, but I suspect that you have fixed this in this PR
<img width="562" alt="image" src="https://github.com/pypeit/PypeIt/assets/12585758/bc9148ed-7a58-49ed-b255-e73d90ba2c5a">